src/tokio/rustls.rs

```diff
@@ -36,6 +36,12 @@ where
                     connector
                 } else {
                     let mut config = ClientConfig::new();
+                    #[cfg(feature = "tokio-rustls-native-certs")]
+                    {
+                        config.root_store =
+                            rustls_native_certs::load_native_certs().map_err(|(_, err)| err)?;
+                    }
+                    #[cfg(feature = "tokio-rustls-webpki-roots")]
```

Comment:
```suggestion
                    #[cfg(all(feature = "tokio-rustls-webpki-roots", not(feature = "tokio-rustls-native-certs"))]
```

Or similar so that it doesn't fall apart when both are enabled. Making native the preferred one seems sensible.